api/v1alpha1/imagerepository_types.go

```diff
@@ -33,6 +33,11 @@ type ImageRepositorySpec struct {
 	// scans of the image repository.
 	// +optional
 	ScanInterval *metav1.Duration `json:"scanInterval,omitempty"`
+
+	// This flag tells the controller to suspend subsequent kustomize executions,
+	// it does not apply to already started executions. Defaults to false.
+	// +optional
+	Suspend bool `json:"suspend,omitempty"`
```

Comment:
In general I'd be surprised to set a field to a meaningful value, then have that field omitted when reported back to me. Here, it's arguable whether `false` and missing are worth distinguishing, since the expected mode of use is to add the flag and _remove_ it, rather than toggling it between `true` and `false`. Since this is (I presume) following the convention used elsewhere in GOTK, I think that argument carries -- but I know future me would appreciate having a comment here about why the general norm is broken, so he doesn't have to think it through again.